.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
```

Comment:
I'll disable dependabot for maven, since we don't want to have incompatible versions suggested. We might enable this for PMD 7 again.

-->

```
open-pull-requests-limit: 0
```